lib/pmdtester/pmd_report_detail.rb

```diff
@@ -25,6 +27,7 @@ def load(report_info_path)
         hash = JSON.parse(File.read(report_info_path))
         @execution_time = hash['execution_time']
         @timestamp = hash['timestamp']
+        @working_dir = hash['working_dir']
```

Comment:
That would expose the working directory again.... which I though, you wanted to remove...

> The baseline is used for sharing, so we should remove this part of the information.

Reply:
This is used to record the working path when generating the pmd report. And we can remove the working dir from pmd report according to value of this 'working dir'